docs/tutorial/01-mlrun-basics.ipynb

```diff
@@ -334,15 +334,15 @@
    "source": [
```

Comment:
<ul><li>Obtain a pandas DataFrame from the&nbsp;<code>source_url</code>&nbsp;data item by calling the&nbsp;<code>as_df</code>&nbsp;method.</li><li>Prepare (clean) the data in the same way as in the local-function implementation in the previous step.</li><li>Record the data length (number of rows) using the&nbsp;<code>log_result</code>&nbsp;method. This method records (logs) the values of standard function variables (for example, int, float, string, and list).</li><li>Log the data-set artifact using the&nbsp;<code>log_dataset</code>&nbsp;method. This method saves and logs function data items and related metadata (logs function artifacts).</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>